build.sh

```diff
@@ -26,4 +26,10 @@ tar cvfpz rpc_so_x64.tar.gz rpc.so.$version
 
 # Add Windows build to a zip file
 zip rpc_windows_x64.zip rpc_windows_x64.exe
-zip rpc_windows_x86.zip rpc_windows_x86.exe
\ No newline at end of file
+zip rpc_windows_x86.zip rpc_windows_x86.exe
+
+# Generate license files
+go-licenses save ./... --include_tests --save_path=licensefiles
```

Comment:
did you read the docs for this flag `--include_tests`? 
>Use the --include_tests global flag to include packages only imported by testing code (e.g., testing libraries/frameworks). 
This is not the behavior we would want. Especially since the tests are not included in the executable. Please remove `--include_tests`

Reply:
resolved